packaging/nnstreamer.spec

```diff
@@ -673,14 +674,14 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
 
 %define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
-%if %{with tizen}
```

Comment:
let change this condition to exclude the tv profile so as to retain the original behavior of running these unittests on all architectures on our CI as discussed offline.

Reply:
Updated.